common/collation/ku.xml

```diff
@@ -13,32 +13,33 @@ For terms of use, see http://www.unicode.org/copyright.html
 	<collations>
 		<!-- TODO: Add collation type="search" -->
 		<collation type="standard" draft="contributed"
-				references="​http://en.wikipedia.org/wiki/Kurdish_alphabets 2013-aug-20">
-			<!-- Initial request and data see cldrbug 6527. -->
+				references="http://en.wikipedia.org/wiki/Kurdish_alphabets 2019-oct-10">
+			<!-- Initial request and data see cldrbug 6527. Subsequent data, see cldrbug 9748. -->
 			<cr><![CDATA[
 				[reorder Latn Arab]
-			
-				# Primary differences for Kurmanji letters.
+
+				# These tailorings are based on the Universal Kurdish Latin Alphabet which seeks
+				# to accomodate all three major dialects: Northern, Central, and Southern Kurdish.
 				&c<ç<<<Ç
 				&e<ê<<<Ê
 				&i<î<<<Î
 				&s<ş<<<Ş
-				&u<û<<<Û
+				&u<û<<<Û<ü<<<Ü
+				&l<ł<<<Ł
+				&n<ň<<<Ň
+				&r<ř<<<Ř
 
-				# Sahand T says: "Éé Íí Úú" are alternative forms
-				# that are used in a variant of the Kurdish-Latin alphabet.
+				# "Éé Íí Úú Ùù" are alternative forms that are used in the Yekgirtú variant of the
+				# Kurdish Latin alphabet.
 				&ê<<é<<<É
 				&î<<í<<<Í
 				&û<<ú<<<Ú
+				&ü<<ù<<<Ù
 
-				# According to Wikipedia, ḧ and ẍ "do not have the status of letter"
-				# which should mean they don't need primary differences.
-				# &h<ḧ<<<Ḧ
-				# &x<ẍ<<<Ẍ
-
-				# Sahand T also suggested tailoring Řř
-				# but that was not requested in cldrbug 6527 nor mentioned in Wikipedia.
-				# &r<ř<<<Ř
+				# The following glyphs are not officially part of the Kurdish alphabet, but are
+				# sometimes used to represent distinct "non-Kurdish" sounds in Kurdish writing.
```

Comment:
This does not mean that they have to have primary differences from the letters without diacritics. The Wikipedia page today still has the "not letter" comment. I suggest keeping this section unchanged.

In other words, the Unicode default sort order already has `h<<ḧ`. Do we need `h<ḧ`?

This might be like in German: äöü represent distinct sounds, but in the standard sort order they have the default secondary differences from their base letters.